src/app/commands/cmd_import_sprite_sheet.cpp

```diff
@@ -314,6 +332,30 @@ class ImportSpriteSheetWindow : public app::gen::ImportSpriteSheet
     }
   }
 
+  void onPaddingEnabledChange() {
+    horizontalPaddingLabel()->setVisible(paddingEnabled()->isSelected());
```

Comment:
Minor change, we can store `paddingEnabled()->isSelected()` in a variable and use it inside the function:
```suggestion
    const bool state = paddingEnabled()->isSelected();
    horizontalPaddingLabel()->setVisible(state);
```